comps/comps-foreman-client-el8.xml

```diff
@@ -15,6 +15,7 @@
       <packagereq type="default">katello-agent</packagereq>
       <packagereq type="default">katello-host-tools</packagereq>
       <packagereq type="default">katello-host-tools-tracer</packagereq>
+      <packagereq type="default">katello-pull-provider-migrate</packagereq>
```

Comment:
Shaving some yaks, IIRC we landed that it is a pull transport rather than provider. Where we are clarifying that "provider" should be used to indicate the user-facing mechanism by which actions are performed (e.g. Script, Ansible) and transport should indicate how the action is delivered. I'd like us to try to be consistent with our use of terminology so I think that would modify this to `katello-pull-transport-migrate` or tie it into REX `katello-remote-execution-pull-migrate` or some such.

@adamruzicka @ezr-ondrej if y'all had a thought

Reply:
I like `katello-pull-transport` and `katello-pull-transport-migrate`! 
KPT™
:+1: